scripts/log-capture

```diff
@@ -61,13 +64,23 @@ _to_log ip r
 _to_log journalctl
 
 cp /etc/resolv.conf ${OUTDIR}
-cp /tmp/*.log ${OUTDIR}
-cp /root/lorax-packages.log ${OUTDIR}
+
+if [[ -e /sys/firmware/efi ]]; then
+    _to_log ls -l /sys/firmware/efi/efivars
+fi
+
+if stat -t /tmp/*.log >/dev/null 2>&1; then
+    cp /tmp/*.log ${OUTDIR}
+fi
+
+if [[ -e /root/lorax-packages.log ]];then
+    cp /root/lorax-packages.log ${OUTDIR}
+fi
 
 if [[ -e /tmp/pre-anaconda-logs/ ]];then
     cp -r /tmp/pre-anaconda-logs ${OUTDIR}
 fi
 
-tar cfa ${ARCHIVE} ${OUTDIR}
+tar cfa ${ARCHIVE} -C / ./${OUTDIR}
```

Comment:
Can you explain why we need to change to root? Wasn't it working before?

Reply:
With the change to root and addition of `./`, `tar` no longer emits:

```
tar: Removing leading `/' from member names
```